{"name": "OrangeHRM Login with Multiple parameters -- @1.2 ", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed\n", "trace": "  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\orangehrmlogin_steps.py\", line 18, in homepage\n    assert False,\"Test Failed\"\n"}, "steps": [{"name": "When enter username \"admin123\" and password \"admin\"", "status": "passed", "start": 1740454963028, "stop": 1740454973295}, {"name": "Then user must be successfully login into the system", "status": "passed", "start": 1740454973295, "stop": 1740454979261}, {"name": "And close browser", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed\n", "trace": "  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\HELLO\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\orangehrmlogin_steps.py\", line 18, in homepage\n    assert False,\"Test Failed\"\n"}, "start": 1740454979261, "stop": 1740454979481}], "parameters": [{"name": "username", "value": "admin123"}, {"name": "password", "value": "admin"}], "start": 1740454952731, "stop": 1740454981590, "uuid": "dbf18618-8eea-4d86-b4f8-2fb9c7b42700", "historyId": "29941d63c525b6a1ca57812033ceb660", "fullName": "OramgeHRM Login: OrangeHRM Login with Multiple parameters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "OramgeHRM Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}